### DISCUSSION TITLE
Theme installation problem --> "Theme error"

### DISCUSSION THREAD

**Discussion by andrewchung** [pradyunsg/furo#608] [Questions] [answered]
Hello -

I have installed the 'furo' theme as per the Quickstart instructions and have come across the following error when building the files:
`Theme error:
no theme named 'furo' found (missing theme.conf?)
`

I have verified that the files have been installed (see attached screenshot) -- however, is there something else that needs to be done?

Using Fedora37, Python 3.11.1

Thanks.
Andy

![Screenshot from 2022-12-30 02-37-14](https://user-images.githubusercontent.com/590100/210046792-c2c05f4d-ede3-454c-8911-28d634cd7a9f.png)

**Comment by pradyunsg** [comment, marked as answer]
You might want to check whether `sphinx-build` is also coming from `./venv/`. It looks like you're on Fedora, so `which sphinx-build` or `where sphinx-build` should give you that information.

**Reply by andrewchung** [reply]
The `sphinx-build` was installed correctly (in `./venv/`).  However, for some unknown reason, PyCharm (the IDE I was testing) didn't recognize it initially.  

Restarting it solved the problem.  Thanks for the pointers!
